dipy/reconst/dki.py

```diff
@@ -889,6 +891,169 @@ def axial_kurtosis(dki_params, min_kurtosis=0, max_kurtosis=3):
     return AK.reshape(outshape)
 
 
+
+
+def _kt_maxima_converge(ang, dt, MD, kt):
+    """ Helper function that computes the negate of the directional kurtosis
```

Comment:
With help from a English native speaking mathematician: I think the word "negate" should be "inverse". I know, that's not how it works with matrices, but apparently that's how it works with numbers.